app/main.py

```diff
@@ -1,2 +1,76 @@
+import copy
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.load_factor = 8
+        self.hash_table: list = [None] * self.load_factor
+
+    def resize_hash_table(self) -> None:
+        outdated_table = copy.copy(self.hash_table)
+        self.load_factor *= 2
+        self.hash_table = [None] * self.load_factor
+        for item in outdated_table:
+            if item:
+                hash_place = item[1] % self.load_factor
+                if not self.hash_table[hash_place]:
+                    self.hash_table[hash_place] = item
+                elif self.hash_table[hash_place]:
+                    while True:
+                        hash_place = (hash_place + 1) % self.load_factor
+                        if not self.hash_table[hash_place]:
+                            break
+                    self.hash_table[hash_place] = item
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
key should be Hashable. Fix everywhere, please
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```